lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -391,6 +393,7 @@ def setUp(self):
         super(ThreadViewSetCreateTest, self).setUp()
         self.url = reverse("thread-list")
 
+    @disable_signal(api, 'thread_created')
```

Comment:
I think I would put the disabling on the class, like you did in the previous test. The reason being that more test cases might be added to the tests, and it might not be clear to the author that they should disable the signals.